lib/ansible/modules/file.py

```diff
@@ -41,7 +41,7 @@
       Set to C(touch) or use the M(ansible.builtin.copy) or M(ansible.builtin.template) module if you want to create the file if it does not exist.
     - If C(hard), the hard link will be created or changed.
     - If C(link), the symbolic link will be created or changed.
-    - If C(touch) (new in 1.4), an empty file will be created if the C(path) does not
+    - If C(touch) (new in 1.4), an empty file will be created if the C(file) does not
```

Comment:
I think C(path) refers to the module option. There is no option `file`, so you can remove the formatting.

Reply:
Thanks. I have just changed it in 8842cc27daac95e98c9819206b7ddc099680bf69